webui/react/src/pages/ModelDetails/ModelHeader.tsx

```diff
@@ -46,8 +47,12 @@ const ModelHeader: React.FC<Props> = ({
     Loaded: (cUser) => cUser.users,
     NotLoaded: () => [],
   }); // TODO: handle loading state
-  const { canDeleteModel } = usePermissions();
-  const { contextHolder, modalOpen } = useModalModelDelete();
+  const { contextHolder: modalModelDeleteContextHolder, modalOpen } = useModalModelDelete();
+  const { contextHolder: modalModelMoveContextHolder, modalOpen: openModelMove } =
+    useModalModelMove();
+  const { canDeleteModel, canModifyModel } = usePermissions();
+  const canDelete = canDeleteModel({ model });
+  const canModify = canModifyModel({ model });
```

Comment:
nit: i think `canDeleteModelFlag` and `canModifyModelFlag` are better naming cuz its hard to guess what the user `canDelete` and `canModify` are (can be model or just description etc).

Reply:
✅